pyiso/caiso.py

```diff
@@ -202,14 +205,17 @@ def get_lmp(self, node_id=None, **kwargs):
             return []
 
         return_list = []
+
+        # revert MARKET_RUN_ID to standardized markets
+        invert_oasis_markets = {v:k for k, v in self.oasis_markets.items()}
         for i, row in df.iterrows():
             dp = {
                 'timestamp': i.to_pydatetime(),  # INTERVALSTARTTIME_GMT is the index
-                'lmp': row['LMP_PRC'],
+                'lmp': row['lmp'],
                 'node_id': row['NODE'],
                 'ba_name': 'CAISO',
                 'lmp_type': row['LMP_TYPE'],
-                'market': self.options['market'],
+                'market': invert_oasis_markets[row['MARKET_RUN_ID']],
```

Comment:
No, there should be.  But calling get_lmp with market='RTPD' failed and the fix was to add the line below (289), or to add market_run_id='RTPD' to the arguments